test/testenv/verificationutils.go

```diff
@@ -569,3 +569,37 @@ func VerifyAppFrameworkDeployment(standalone *enterprisev1.Standalone) {
 	}
 
 }
+
+// VerifyAppInstall verify that app of specific version is installed
+func VerifyAppInstall(deployment *Deployment, testenvInstance *TestEnv, ns string, podName string, apps map[string]string, versionCheck bool, statusCheck string) {
+	for appName, version := range apps {
+		status, versionInstalled, err := GetPodAppStatus(deployment, podName, ns, appName)
+		gomega.Expect(err).Should(gomega.Equal(""))
+		gomega.Expect(status).Should(gomega.Equal(statusCheck))
+		if versionCheck {
+			gomega.Expect(versionInstalled).Should(gomega.Equal(version))
+		}
+	}
+}
+
+// VerifyAppsCopied verify that apps are copied to correct location based on CR spec
+func VerifyAppsCopied(deployment *Deployment, testenvInstance *TestEnv, ns string, podName string, apps []string) {
+	path := "etc/apps"
+	if strings.Contains(podName, "cluster-master") {
+		path = "etc/master-apps/_cluster/default"
```

Comment:
Should this be `etc/master-apps` instead of `_cluster/default`?

Reply:
when running locally it seemed to be under this path. We can modify based on tests